templates/jasmine/jasmine-templates.js

```diff
@@ -0,0 +1,19 @@
+// Utility templates
+var tempRequire = 'var {{=it.varName}} = require(\'{{=it.module}}\');';
+
+// Base template
+var baseTemplate = 'describe(\'{{=it.testTitle}}\', function() { ';
+
+// Individual assertion templates
+var equalTemplate = 'it(\'{{=it.assertionMessage}}\', function() {\nexpect(file.{{=it.assertionInput}}{{=it.assertionType}}({{=it.assertionOutput}}))\n});';
+var notEqualTemplate = 'it(\'{{=it.assertionMessage}}\', function() {\nexpect(file.{{=it.assertionInput}}{{=it.assertionType}}({{=it.assertionOutput}}))\n});';
+
+module.exports = {
+  require: tempRequire,
+  base: baseTemplate,
+  equal: equalTemplate,
```

Comment:
@zds97 and I discussed this a bit earlier, but we may want to get a consensus on the naming of the templates, specifically ```equal``` vs ```equals```. Not a problem in this case, as Jasmine uses ```equal```, but Tape does use ```equals```.

![image](https://cloud.githubusercontent.com/assets/3118710/9674862/dab628fc-526a-11e5-82fd-6971d46e5be5.png)